map.js

```diff
@@ -101,9 +123,95 @@ function zoomMap(event){
     if(event.deltaY > 0) zoomLevel += 0.2;
     else zoomLevel += -0.2;
     
+    //TODO: make it so that it zooms into middle of screen rather than top left corner?
+
```

Comment:
do we want to have the map drawn from the center of the screen? does drawing it from the upper left corner cause issues anywhere else?

Reply:
this is the only issue i've had with drawing from the top left, but it also can be overcome i think with a bit more math application in the zoomap function.